test/realm_test.dart

```diff
@@ -1864,7 +1864,7 @@ Future<void> main([List<String>? args]) async {
     var config = Configuration.local([Car.schema], path: "${generateRandomUnicodeString()}.realm");
     var realm = getRealm(config);
     expect(realm.isClosed, false);
-  });
+  }, skip: Platform.isAndroid || Platform.isIOS);
```

Comment:
Why is this needed?

Reply:
[test_api-0.5.0 ](https://pub.dev/packages/test_api) was released 3 days ago.

Reply:
Shold we create a task to investigate and remove this later?

Reply:
Also if it fails on Android why disable for iOS?

Reply:
The tests fail for both [Android](https://github.com/realm/realm-dart/actions/runs/4531527657/jobs/7982684657#step:9:1215) and [IOS ](https://github.com/realm/realm-dart/actions/runs/4531527657/jobs/7982684404#step:9:2484) when having unicode symbols in the realm path. I will disable these tests in order to be able to fix the main branch. I [created a new issue for the paths.](https://github.com/realm/realm-dart/issues/1230)